lib/seqset.c

```diff
@@ -139,6 +140,46 @@ EXPORTED void seqset_add(seqset_t *seq, unsigned num, int ismember)
     seq->prev = num;
 }
 
+/*
+ * Remove a number `num' from the sequence set `seq'.
+ * Currently assumes that it will be called in
+ * monotonically increasing order of `num's.
```

Comment:
This I find problematic. At least the code should assert that the input comes in the right order?

Reply:
Yeah! Sorry for the noise.